libs/execution_context/unit/execution_context_unit_test.cpp

```diff
@@ -124,13 +124,16 @@ _test_crud_operations(ebpf_map_type_t map_type)
     bool is_array;
     bool supports_find_and_delete;
     bool replace_on_full;
+    bool insert_on_full = false;
```

Comment:
Why is this one initialized to false but replace_on_full isn't?

Reply:
`replace_on_full` is being explicitly initialized for each map type below.